packages/discv5/README.md

```diff
@@ -14,29 +14,26 @@ Included is a libp2p peer-discovery compatibility module.
 #### Example
 
 ```typescript
-import { Discv5Discovery, ENR } from "@chainsafe/discv5";
-import Libp2p from "libp2p";
+import {Discv5Discovery} from "@chainsafe/discv5";
+import {ENR, SignableENR} from "@chainsafe/enr";
```

Comment:
do we have to import these from enr package or docs discv5 report? the later seems better devex to me

Reply:
I haven't seen them being re-exported. Maybe @wemeetagain knows better?